mne_qt_browser/_pg_figure.py

```diff
@@ -64,6 +63,66 @@ def capture_exceptions():
 name = 'pyqtgraph'
 
 
+# This can be removed when mne==1.0 is released.
+try:
+    from mne.viz.backends._utils import _init_mne_qtapp
+except ImportError:
```

Comment:
Or I guess for completeness
```suggestion
except (ImportError, ModuleNotFoundError):
```

Reply:
```py
In [1]: issubclass(ModuleNotFoundError, ImportError)
Out[1]: True
```

Reply:
Nice, went back to plain `ImportError` then